src/physics/em/detail/MollerEnergyDistribution.i.hh

```diff
@@ -21,15 +21,15 @@ namespace detail
  */
 CELER_FUNCTION
 MollerEnergyDistribution::MollerEnergyDistribution(
-    const MollerBhabhaPointers& shared, const real_type inc_energy)
-    : electron_mass_c_sq_(shared.electron_mass_c_sq)
-    , inc_energy_(inc_energy)
-    , total_energy_(inc_energy + shared.electron_mass_c_sq)
-    , max_energy_fraction_(0.5)
-    , min_energy_fraction_(shared.min_valid_energy / inc_energy)
-    , gamma_(total_energy_ / shared.electron_mass_c_sq)
+    const real_type electron_mass_c_sq,
```

Comment:
Even though I said that it's better to make variables `const` if they're not being modified, I avoid making function arguments `const` (previously this was a reference to const data, so it got left in). In the Babhba model the constructor also have a couple extra `const`s.

Reply:
> I avoid making function arguments const 
Why do you avoid it?  It should allow the compiler do a better job in optimizing the function, right?

Reply:
@mrguilima It won't if they're local variables (which is the case since they're function arguments). See https://github.com/celeritas-project/celeritas/wiki/Development#function-arguments-and-return-values . The constness of local variables generally won't help smart compilers, since they know the scope of the variable and can deduce whether it's modified (or whether a reference to it could result in a change in its value through a const-cast, for example). In the original code that @stognini ported, there were a dozen variables, many of which were temporary but never modified after creation, and a few of them were expected to be modified. The `const` there was more of a hint to us humans that it's not going to be changed.